contracts/zero-ex/contracts/src/features/NativeOrdersFeature.sol

```diff
@@ -346,7 +345,6 @@ contract NativeOrdersFeature is
                 taker: taker,
                 sender: sender
             }));
-        _refundExcessProtocolFeeToSender(results.ethProtocolFeePaid);
```

Comment:
This is a no-op if `msg.sender == address(this)`, and this function is `onlySelf`

Reply:
this was the only functional change to this feature, so no need to redeploy